### DISCUSSION TITLE
how to determine the texture components {hkl} <uvw> from a given pole figure?

### DISCUSSION THREAD

**Discussion by jucego333** [mtex-toolbox/mtex#1039] [Ask Anything]
Hello dear colleagues. 

I have been working with my EBSD Data and I successfully plotted the pole figure (see left picture). Now, I would like to know how to determine the texture components (see right figure). 

I have checked the section of Standard Orientations (https://mtex-toolbox.github.io/OrientationStandard.html), but I don't know how to define "components ". Another way that I found is the conversion of Euler orientation to corresponding miller indices, (https://github.com/mtex-toolbox/mtex/discussions/956), but I cannot find an example of how to use orientation/round2Miller prompt. Could anybody give me a hand? Thanks

![image](https://user-images.githubusercontent.com/78469467/118388949-f1341900-b659-11eb-8e0d-e80b68fefdab.png)

**Comment by DriptD** [comment]
Hi,

How about this - 

ori = orientation.[byEuler](https://mtex-toolbox.github.io/orientation.byEuler.html#3)(phi1,Phi,phi2,CS,SS)
[uvw,hkl] = [round2Miller](https://mtex-toolbox.github.io/orientation.round2Miller.html#4)(ori)



Cheers

**Reply by jucego333** [reply]
@DriptD  One more time thank you for your quick reply. I can try to analyze in this way.

**Reply by jucego333** [reply]
@DriptD   I have a question when I prompt the following
fff = orientation.byEuler(259.2,159.2, 255.3,cs,ss)  (taking from the picture)
I obtained the following :

Bunge Euler angles in degree
     phi1     Phi    phi2    Inv.
  294.592  15.211 32.9578       0

I don't understand why these values differ from 259.2,159.2, 255.3.  what do they mean?

**Reply by jucego333** [reply]
I got it , I should use   fff= orientation.byEuler(259.2*degree,159.5*degree,255.3*degree,cs,ss). Thanks

**Reply by DriptD** [reply]
Hi,

Could you please try with  `orientation.byEuler(259.2*degree,159.2*degree,255.3*degree,cs,ss)`

Cheers
Dripta
